projects/experimental/components/card/card.component.ts

```diff
@@ -0,0 +1,34 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    HostListener,
+    Inject,
+    Input,
+} from '@angular/core';
+import {TUI_PLATFORM, TuiPlatform} from '@taiga-ui/cdk';
+
+@Component({
+    selector: 'tui-card',
+    templateUrl: './card.template.html',
```

Comment:
Let's make it a directive without the template. All the positioning options can be easily done with flex styles on the host. This way people can use it on buttons or links. Don't forget clearbtn and text-decoration on host.

Reply:
fixed